tutorials/interop-with-pandas-and-polars.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Interoperability with Pandas and Polars
+layout: default
+nav_order: 2
+parent: Tutorials
+---
+
+GlareDB can seamlessly read from, and write to, Panda and Polars. This enables
+many interesting workflows for data exploration as well as integration with
+other data tools.
+
+## Select from a Pandas or Polars dataframe with GlareDB
+
+If you have a Pandas dataframe in memory, you can readily select from
+it:
+
+```python
+import pandas as pd
+import glaredb
+
+df = pd.read_csv('https://github.com/GlareDB/glaredb/raw/main/testdata/csv/userdata1.csv')
+
+con = glaredb.connect()
+con.sql(
+    """
+        SELECT * FROM df WHERE country='Canada'
+    """
+).show()
+```
+
+Notice how the `df` variable to which the Pandas dataframe is assigned is
+included in the session scope, and so can be dropped into the query as if
+it were a table in the database.
```

Comment:
I think we should omit the wording of "Notice" and "dropped" to avoid a tone in the former case and confusion with dropping tables in the latter.

Something like:

```MD
The `df` variable, a Pandas dataframe, is in scope and be referenced in the GlareDB query as if it were a table in a database.
```